File: py/2770.py
while True:
    try:
        x, y, m = map(int, input().split())

        for _ in range(m):
            x2, y2 = map(int, input().split())

            if (x2 <= x and y2 <= y) or (x2 <= y and y2 <= x):
                print('Sim')

            else:
                print('Nao')

    except EOFError:
        break

File: cpp/2770.cpp
#include <iostream>

using namespace std;

int main() {
    int x, y, m;

    while (scanf("%d %d %d", &x, &y, &m) != EOF) {
        int i, x2, y2;

        for (i = 0; i < m; i++) {
            scanf("%d %d", &x2, &y2);

            if ((x2 <= x && y2 <= y) || (x2 <= y && y2 <= x)) {
                cout << "Sim" << endl;
            } else {
                cout << "Nao" << endl;
            }
        }
    }
}